File: lua/teste_cpf.lua
#!/usr/bin/env lua

local cpf = require "valida_cpf";

if ( arg[1] == "-h" ) then
    cpf.print_usage();
else
    if ( cpf.valida_cpf( arg[1] ) ) then
        cpf.formata( arg[1] );
    else
        print("Falha: cpf "..arg[1].." é inválido.");
    end
end

File: python/teste_cpf.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
import documentos.cpf
import sys

__cpf =  sys.argv[1]

if documentos.cpf.valida(__cpf):
    print "valido"
    print documentos.cpf.formata(__cpf);
else:
    print "nao valido";
